packages/bits/.percy.yml

```diff
@@ -0,0 +1,29 @@
+version: 2
+snapshot:
+  # An array of numbers, in pixels, representing the widths you'd like to capture for each snapshot
+  widths:
+    - 1920
+  # A number specifying the minimum height in pixels each snapshot should be
+  minHeight: 1080
+  # A string containing Percy specific CSS that will be applied to each snapshot
+  # (per-snapshot Percy CSS is concatenated to the end of this value)
+  # Details: https://docs.percy.io/docs/percy-specific-css
+  percyCSS: |
+    input, textarea { 
+      caret-color: transparent;
+    }
+    * {
+      cursor: none !important;
+    }
+  # WE DON'T CARE ABOUT THE CONFIGS BELOW
```

Comment:
Just curious, why are they here if we don't care about them? Maybe this information could be included in the comment?

Reply:
Theoretically, those options could be used for another way of taking snapshots - taking local screenshots and bulk-sending them to percy. I decided to leave them there for future reference, I think I'll be reviewing it in the next sprint in the scope of other percy tasks, so no worries.